src/schemas/json/dependabot-2.0.json

```diff
@@ -1132,6 +1156,130 @@
+              "pattern": "^([01][0-9]|2[0-3]):[0-5][0-9]$"
+            },
+            "timezone": {
+              "$ref": "#/definitions/timezone",
+              "description": "The time zone identifier must be from the Time Zone database maintained by IANA"
+            },
+            "cronjob": {
+              "type": "string",
+              "description": "Specify a valid cron expression for updates"
+            }
+          },
+          "allOf": [
+            {
+              "$comment": "If interval type is 'cron', enforce 'cronjob' property.",
+              "if": {
+                "properties": {
+                  "interval": {
+                    "const": "cron"
+                  }
+                }
+              },
+              "then": {
+                "required": ["interval", "cronjob"]
+              },
+              "else": {
+                "required": ["interval"]
+              }
+            }
+          ]
+        },
+        "labels": {
+          "description": "Labels to set on pull requests (additive - merges with ecosystem-level labels)",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "minItems": 0,
+          "uniqueItems": true
+        },
+        "assignees": {
+          "description": "Assignees to set on pull requests (additive - merges with ecosystem-level assignees)",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "milestone": {
+          "description": "Associate all pull requests raised for this group with a milestone. You need to specify the numeric identifier of the milestone and not its label.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "target-branch": {
+          "description": "Specify a different branch for manifest files and for pull requests.",
+          "type": "string",
+          "minLength": 1
+        },
+        "commit-message": {
+          "description": "Commit message preferences for the group",
+          "type": "object",
+          "properties": {
+            "prefix": {
+              "description": "A prefix for all commit messages",
+              "type": "string",
+              "maxLength": 50
+            },
+            "prefix-development": {
+              "description": "A separate prefix for all commit messages that update dependencies in the Development dependency group",
+              "type": "string",
+              "maxLength": 50
+            },
+            "include": {
+              "description": "Specifies that any prefix is followed by a list of the dependencies updated in the commit",
+              "type": "string",
+              "enum": ["scope"],
+              "default": "scope"
```

Comment:
While only the value `"scope"` is supported _when `include` is defined_, I don't think that it has a default value of `"scope"`.

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#include